p2p/src/protocol.rs

```diff
@@ -85,6 +85,8 @@ impl Link {
 /// A command or request that can be sent to the protocol.
 #[derive(Debug, Clone)]
 pub enum Command {
+    /// Get block height.
+    GetBlockByHeight(u64, chan::Sender<Option<BlockHash>>),
```

Comment:
Use `Height` here for the type. Also, I think this command would be more useful if it returned the `BlockHeader` at that height.

Reply:
Fixed!